server/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -29,7 +29,7 @@
  */
 public abstract class AggregationBuilder
     implements
-        NamedWriteable,
+        VersionedNamedWriteable,
```

Comment:
why adding the method in another class, if this is the class that implements the versioned variant?